ui/app/src/components/CreateUserDialog/CreateUserDialogContainer.tsx

```diff
@@ -51,6 +51,7 @@ import {
 } from '../UserManagement/utils';
 import useUpdateRefs from '../../hooks/useUpdateRefs';
 import { showSystemNotification } from '../../state/actions/system';
+import zxcvbn from 'zxcvbn';
```

Comment:
This is a large lib which will unnecessarily grow the bundle. We need a async, on-demand mechanism to load it.

Reply:
still pending